configuration.go

```diff
@@ -0,0 +1,18 @@
+package codacytool
+
+import (
+	"github.com/josemiguelmelo/gofile"
```

Comment:
why are we depending on a personal project?
can we use the standard library?

Reply:
I replaced it with the implementation of a utility to read and parse the json file